kubernetes/argocd/stacks/sparkhacks/sparkhacks-2023.yml

```diff
@@ -68,7 +68,7 @@ spec:
       match: "Host(`sparkhacksatuic.com`) || Host(`www.sparkhacksatuic.com`)"
       services:
         - kind: Service
-          name: sparkhacks-2023
+          name: sparkhacks-2024 # Perhaps move to new file.
```

Comment:
Yeah, we can move this to a different file. I was thinking when we switch, I'd add a `/2023` IngressRoute, so the old site can still be accessible.